src/oc/hoc_init.cpp

```diff
@@ -222,6 +222,7 @@ static struct { /* Builtin functions with multiple or variable args */
                  {"mcell_ran4_init", hoc_mcran4init},
                  {"nrn_feenableexcept", nrn_feenableexcept},
                  {"nrnmpi_init", hoc_nrnmpi_init},
+                 {"coreneuron_handle", hoc_coreneuron_handle},
```

Comment:
This can be used to trigger loading the CoreNEURON library from HOC/Python, which will be used in https://github.com/neuronsimulator/nrn/pull/1922.